libs/initScripts/helpers/version.js

```diff
@@ -6,26 +6,49 @@ const web3 = require('web3')
  * @param {String} serviceType service type trying to register
  * @param {String} serviceVersionStr version string to register
  * @param {String?} privateKey optional private key string
+ * @param {Boolean?} dryRun Optional parameter to return the generated parameters without sending tx
  */
-async function setServiceVersion (audiusLibs, serviceType, serviceVersionStr, privateKey = null) {
+async function setServiceVersion (audiusLibs, serviceType, serviceVersionStr, privateKey = null, dryRun = false) {
```

Comment:
dryRun is also default to false here so it's backwards compatible. other scripts like local.js won't be affected by this change